### ISSUE TITLE
Create titlebank folders within standalone process

### ISSUE THREAD

**Issue by HierToo** [TheGameCreators/GameGuruRepo#658]
Hi,

it would be a great feature if GG would check which resolutions are in the titlebank folder and if some is missing, it should create the folder from a higher resolution folder with the correct aspect ration.
Also some new resolutions could be added to support newer hardware.

Resolution 	| AspectRatio
1280x720	| 16:9	
1280x800	| 16:10
1366x768	| 16:9
1440x900	| 16:10
1600x900	| 16:9
1680x1050	| 16:10
1920x1080	| 16:9
1920x1200	| 16:10
New resolutions
2560 × 1440	| 16:9
(4K) 3840 x 2160| 16:9
(4K) 4096 x 2160| 1.9:1

Thanks

**Comment by michaeloldman** [comment]
Yeah agree.
And also don't use anything from the languagebank folder.
Standalone Screens should all be sourced ONLY from the titlesbank folder.

OldFlak....

**Comment by synchromesh62** [comment]
Agreed on the Watermark screens contained in In Languagebank \ English \ artwork. ( posted this also )
Any chance this could be disabled or removed as it just another 12 screens we have to edit on top of the already 40 screens we need to edit \ resize for our menu's.
They really don't serve any purpose and are just extra work for us to get the professional look :(

**Comment by Belidos3D** [comment]
Or instead of having a bajillion different resolutions to bloat the game, have a high res image for each of the three or four common aspect ratios, and then scale based on aspect ratio.
